rules/typescript.js

```diff
@@ -2,23 +2,34 @@ module.exports = {
   plugins: ['@typescript-eslint'],
   extends: ['plugin:@typescript-eslint/recommended'],
   rules: {
-    "@typescript-eslint/ban-ts-ignore": "off",
+    '@typescript-eslint/ban-ts-ignore': 'off',
     '@typescript-eslint/array-type': 'error',
     '@typescript-eslint/explicit-function-return-type': 'off',
-    '@typescript-eslint/member-delimiter-style': "off",
+    '@typescript-eslint/consistent-type-definitions': ['error', 'type'],
```

Comment:
There may be a couple valid cases for when we may need interfaces (types and interfaces are not fully equivalent).
But I guess it is fine for now. We may not even even run into such situations any time soon. And if we do we can discuss this rule later.